components/modal/borrow-more-modal.tsx

```diff
@@ -34,59 +41,111 @@ const BorrowMoreModal = ({
   position,
   onClose,
 }: {
-  position: {
-    currency: Currency
-    amount: string
-  } | null
+  position: LoanPosition
   onClose: () => void
 }) => {
+  const { prices } = useCurrencyContext()
   const [value, setValue] = useState('')
+
+  const amount = useMemo(
+    () => (position ? parseUnits(value, position.underlying.decimals) : 0n),
```

Comment:
Let's
```
if (!position) {
    return <></>
}
```
And get rid of all null checks for the code below

Reply:
<img width="567" alt="image" src="https://github.com/clober-dex/coupon.finance/assets/10525011/02decd08-a49b-4b8f-881e-0e1589a714ea">

I believe setting the 'amount' to zero would make it cleaner. Additionally, it appears that the withdrawal model logic is the same. You can refer to it here: [Link](https://github.com/clober-dex/coupon.finance/blob/dev/components/modal/withdraw-modal.tsx#L27)

Reply:
Isn't position never null?

Reply:
Yeah!